src/traces/isosurface/fn_find-nearest-on-axis.js

```diff
@@ -0,0 +1,29 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var findNearestOnAxis = function(w, arr) {
```

Comment:
I would prefer if `findNearestOnAxis` and `generateIsoMeshes` were brought back in `isosurface/convert.js`.

To reuse them in `volume/convert.js`, you'll need to turn

```js
module.exports = createIsosurfaceTrace;
```

into 

```js
module.exports = {
  createIsosurfaceTrace: createIsosurfaceTrace,
  findNearestOnAxis: findNearestOnAxis,
  generateIsoMeshes: generateIsoMeshes
}
```

in `isosurface/convert.js` and then `require('../isosurface/convert')` in `volurme/convert.js`